lib/identity_config.rb

```diff
@@ -136,6 +136,7 @@ def self.build_store(config_map)
     config.add(:async_wait_timeout_seconds, type: :integer)
     config.add(:attribute_encryption_key, type: :string)
     config.add(:attribute_encryption_key_queue, type: :json)
+    config.add(:available_locales, type: :json)
```

Comment:
IMO, JSON is more typo–prone than comma-separated?

Reply:
yeah, good point. https://github.com/18F/identity-idp/pull/10346 was still kind of stuck in my head, but we'll (hopefully) never need commas in the values here.